src/ui/server/helpers/get-submitted-fields/index.ts

```diff
@@ -0,0 +1,43 @@
+import isPopulatedArray from '../is-populated-array';
+import { ApplicationFlat } from '../../../types';
+import isFieldFalseOrZero from '../is-field-false-or-zero';
+
+/**
+ * hasSubmittedField
+ * @param {Object} submittedData Submitted application data
+ * @param {String} field ID of the field to get
+ * @returns {Boolean} True if the field is in submittedData.
+ */
+// Note: this assumes that any data in submitted fields is a valid answer. E.g, false boolean is a valid answer.
+export const hasSubmittedField = (submittedData: ApplicationFlat, fieldId: string) => {
+  // if array, check it is not empty array
+  if (submittedData && fieldId && isPopulatedArray(submittedData[fieldId])) {
+    return true;
+  }
+
+  if (submittedData && fieldId && (submittedData[fieldId] || isFieldFalseOrZero(submittedData[fieldId])) && !Array.isArray(submittedData[fieldId])) {
```

Comment:
Can the compound condition please be in separate lines since it is over three?

Reply:
Made it simpler and easier to read 👍